[PATCH] Do not trap access to timer and keyboard

Some VESA BIOSes need to access to them.

Signed-off-by: Samuel Thibault <samuel.thibault@ens-lyon.org>
---
 hw/xfree86/os-support/hurd/hurd_video.c |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/hw/xfree86/os-support/hurd/hurd_video.c b/hw/xfree86/os-support/hurd/hurd_video.c
index 4a99db3..e049ceb 100644
--- a/hw/xfree86/os-support/hurd/hurd_video.c
+++ b/hw/xfree86/os-support/hurd/hurd_video.c
@@ -124,8 +124,6 @@ xf86EnableIO()
 	FatalError("xf86EnableIO: ioperm() failed (%s)\n", strerror(errno));
 	return FALSE;
     }
-    ioperm(0x40,4,0); /* trap access to the timer chip */
-    ioperm(0x60,4,0); /* trap access to the keyboard controller */
     return TRUE;
 }
 	
